core/src/workflows/hold_link.rs

```diff
@@ -76,10 +76,11 @@ pub mod tests {
             create_test_dna_with_wat("test_zome", "test_cap", Some(&test_wat_always_invalid()));
         dna.uuid = String::from("test_reject_invalid_link_on_hold_workflow");
 
-        // Hash of the original DNA
-        let dna_hash = base64::encode(&dna.multihash().unwrap());
+        // Address of the original DNA
+        let dna_address = dna.address();
```

Comment:
much better to have this consistent function than calling base64... repeatedly. Sweet!

Reply:
@Connoropolous yar, based on the experience where we decided Q3 last year to change the way addresses work for agents, i think it's pretty important to leverage `.address()` as much as possible - it keeps this as flexible as possible